[2022-12-11 19:22:32,955] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_DAG.run_etl_process scheduled__2022-12-08T15:00:00+00:00 [queued]>
[2022-12-11 19:22:33,024] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_DAG.run_etl_process scheduled__2022-12-08T15:00:00+00:00 [queued]>
[2022-12-11 19:22:33,027] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 19:22:33,029] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-12-11 19:22:33,030] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 19:22:33,156] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): run_etl_process> on 2022-12-08 15:00:00+00:00
[2022-12-11 19:22:33,181] {standard_task_runner.py:52} INFO - Started process 390 to run task
[2022-12-11 19:22:33,226] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_DAG', 'run_etl_process', 'scheduled__2022-12-08T15:00:00+00:00', '--job-id', '230', '--raw', '--subdir', 'DAGS_FOLDER/etl_DAG.py', '--cfg-path', '/tmp/tmpqb_pln_4', '--error-file', '/tmp/tmp5_1opt3l']
[2022-12-11 19:22:33,231] {standard_task_runner.py:80} INFO - Job 230: Subtask run_etl_process
[2022-12-11 19:22:33,684] {task_command.py:369} INFO - Running <TaskInstance: etl_DAG.run_etl_process scheduled__2022-12-08T15:00:00+00:00 [running]> on host 0163e0790208
[2022-12-11 19:22:34,301] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_DAG
AIRFLOW_CTX_TASK_ID=run_etl_process
AIRFLOW_CTX_EXECUTION_DATE=2022-12-08T15:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-08T15:00:00+00:00
[2022-12-11 19:22:41,231] {python.py:173} INFO - Done. Returned value was: None
[2022-12-11 19:22:41,308] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_DAG, task_id=run_etl_process, execution_date=20221208T150000, start_date=20221211T192232, end_date=20221211T192241
[2022-12-11 19:22:41,438] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-11 19:22:41,658] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
